frontend/src/main/components/Users/UsersTable.js

```diff
@@ -29,11 +31,49 @@ const columns = [
         id: 'admin',
         accessor: (row, _rowIndex) => String(row.admin) // hack needed for boolean values to show up
     },
+    {
+        Header: 'Instructor',
+        id: 'instructor',
+        accessor: (row, _rowIndex) => String(row.instructor) // hack needed for boolean values to show up
+    }
 ];
 
-export default function UsersTable({ users }) {
+export default function UsersTable({ users, showToggleButtons }) {
+    
+    // toggle admin
+    // Stryker disable all : hard to test for query caching
+    const toggleAdminMutation = useBackendMutation(
+        cellToAxiosParamsToggleAdmin,
+        { onSuccess: toggleAdminSuccess},
+        ["/api/admin/users/all"]
```

Comment:
I think this is the bug that's causing the page not to update when you toggle admin and instructor statuses.
```suggestion
        ["/api/admin/users"]
```

This parameter is the cache key to invalidate.  Invalidating the cache key causes the query be done again, and the component to render again.  We want the users table to render again after we toggle the admin status or instructor status because the data has changed.

Compare the url that is being used to get the users which comes from: 
`frontend/src/main/utils/users.js`

```js
export function useUsers() {
  return useQuery("users", async () => {
    const uri = "/api/admin/users";
    try {
      const response = await axios.get(uri);      
      return response.data ;
    } catch (e) {
      console.error(`Error getting data from ${uri}:`,e);
      return [];
    }
  }, {
    initialData: []
  });
}
```

That url is `/api/admin/users`  so THAT is the key that we need to invalidate.

Reply:
So the url here is the cache key to invalidate, that makes so much sense